transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java

```diff
@@ -33,8 +33,8 @@
  * it only works on linux.
  */
 public final class EpollEventLoopGroup extends MultithreadEventLoopGroup {
-    static {
-        // Ensure JNI is initialized by the time this class is loaded by this time!
+    {
```

Comment:
static is called when the class is loaded (once, when an object is instantiated or you call a static method), instance initializer is not invoked when the class is loaded (will not be invoked if you invoke a static method but will be invoked every time you instantiate an object of that class).